Add the cros-specific toolchain names to the sanitizer_lib_boilerplate
function so that asan etc. function correctly for all x86_64-*-linux targets.

diff --git a/src/build_helper/lib.rs b/src/build_helper/lib.rs
index bd99dc118e..bf1a3ffee0 100644
--- a/src/build_helper/lib.rs
+++ b/src/build_helper/lib.rs
@@ -271,7 +271,9 @@ pub fn sanitizer_lib_boilerplate(sanitizer_name: &str)
     -> Result<(NativeLibBoilerplate, String), ()>
 {
     let (link_name, search_path, apple) = match &*env::var("TARGET").unwrap() {
-        "x86_64-unknown-linux-gnu" => (
+        "x86_64-unknown-linux-gnu"
+        | "x86_64-cros-linux-gnu"
+        | "x86_64-pc-linux-gnu" => (
             format!("clang_rt.{}-x86_64", sanitizer_name),
             "build/lib/linux",
             false,
